### PR TITLE
traceroute: update page

### PR DESCRIPTION
**Pull request by marek22k**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** `Modern traceroute for Linux, version 2.1.0`

### CODE REVIEW

File: pages/common/traceroute.md [hunk 1]
```diff
@@ -5,20 +5,28 @@
...
 
 - Disable IP address and host name mapping:
 
-`traceroute -n {{host}}`
+`traceroute -n {{example.com}}`
 
 - Specify wait time for response:
 
-`traceroute -w {{0.5}} {{host}}`
+`traceroute --wait={{0.5}} {{example.com}}`
```
Comment (mfrw) [inline pages/common/traceroute.md:16]: Is this wait time in seconds ? I think we should update the unit as well

Comment (marek22k) [reply pages/common/traceroute.md:16]: >   -w MAX,HERE,NEAR  --wait=MAX,HERE,NEAR
                              Wait for a probe no more than HERE (default 3)
                              times longer than a response from the same hop,
                              or no more than NEAR (default 10) times than some
                              next hop, or MAX (default 5.0) seconds (float
                              point values allowed too)

Yes, is in seconds.

Comment (marek22k) [reply pages/common/traceroute.md:16]: I have now written the unit to it.

**Review by kbdharun — APPROVED** [review]

**Comment by marek22k** [comment]
Is there a reason for the closing and reopening?

**Comment by kbdharun** [comment]
> Is there a reason for the closing and reopening?

The closing and reopening of PRs are done when the CLA check is stuck at the "waiting for status" message, doing this will trigger rechecking the CLA status and if you have signed it already it would succeed or else there would be a message asking you to sign the CLA.

**Comment by mfrw** [comment]
> > Is there a reason for the closing and reopening?
> 
> The closing and reopening of PRs are done when the CLA check is stuck at the "waiting for status" message due to some issue with GitHub API, doing this will trigger rechecking the CLA status and if you have signed it already it would succeed or else there would be a comment asking you to sign the CLA.

Today I learnt .. how to retrigger the CLA :)
I did not know this

**Review by mfrw — APPROVED** [review]
LGTM :)
Thank you for your contribution @marek22k

**Pull request merged**